src/imu_vn_100.cpp

```diff
@@ -294,6 +298,31 @@ void ImuVn100::CreatePublishers() {
   }
 }
 
+void ImuVn100::CreateSubscribers() {
+  if (ref_use_models_) {
+    using std::placeholders::_1;
```

Comment:
Nit: since this is only used one place, I'd prefer to just use it in the line below.  But I'll leave it up to you whether to change that.

Reply:
Sounds good. Done in 321aff1f5c3eff46c4dc3a98e1dbab586690963e.